src/mc_tmpl_8.rs

```diff
@@ -2037,7 +2037,7 @@ unsafe extern "C" fn put_8tap_regular_c(
         mx as usize,
         my as usize,
         DAV1D_FILTER_8TAP_REGULAR | (DAV1D_FILTER_8TAP_REGULAR << 2),
-        BitDepth8::new(()),
+        BitDepth8::new(8),
```

Comment:
We don't need to add this I don't think.  `trait BitDepth` already has `fn bitdepth` and `fn bitdepth_max` methods.